### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by wuyepabdul**
In this exercise, I did the following:
    • Setup webpack
    • Added index.html, index.js, and style.css files
    • Added markup to index.html
    • Added css styling to style.css

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import './style.css';
```
Comment (helman101) [inline src/index.js:1]: - [x] At this point of the project you should have developed a working version of the application as the project requirements said. Kindly take a look at this screenshot and to project requirements. 😸 
    - All the buttons should be functional, which means, the submit button should add a score to the list, and the refresh button should refresh the list of scores 🚀
    - Remember to use ES6 syntax
    imagen

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @wuyepabdul  👋, 

You have done a great job with the template of your page,  👏👏👏 
but there are still some changes that you have to do.

## To Highlight 🌟 

- Nice job on the page template 😄 
- Nice job setting up webpack 🚀 

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (wuyepabdul) [reply src/index.js:1]: Please kindly find attached a copy of the project requirements. We are to implement the changes you requested in the next project milestone. 
requirements

Comment (Emmyn5600) [reply src/index.js:1]: I see you will implement the following tasks in the next milestone.

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @wuyepabdul 


## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
 - N/A.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)Hi,

**Pull request merged**